{
	"schema_version": "1.6.0",
	"id": "CVE-2018-8099",
	"modified": "2024-07-31T01:31:20.201553Z",
	"published": "2018-03-14T00:29:00Z",
	"related": [
		"DLA-2936-1"
	],
	"details": "Incorrect returning of an error code in the index.c:read_entry() function leads to a double free in libgit2 before v0.26.2, which allows an attacker to cause a denial of service via a crafted repository index file.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "libgit2",
				"purl": "pkg:deb/debian/libgit2?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.27.0+dfsg.1-0.6"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "low"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-8099.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "libgit2",
				"purl": "pkg:deb/debian/libgit2?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.27.0+dfsg.1-0.6"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "low"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-8099.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "libgit2",
				"purl": "pkg:deb/debian/libgit2?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.27.0+dfsg.1-0.6"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "low"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-8099.json"
			}
		},
		{
			"package": {},
			"ranges": [
				{
					"type": "GIT",
					"repo": "https://github.com/libgit2/libgit2",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "58a6fe94cb851f71214dbefac3f9bffee437d6fe"
						}
					]
				}
			],
			"versions": [
				"v0.1.0",
				"v0.10.0",
				"v0.11.0",
				"v0.12.0",
				"v0.13.0",
				"v0.14.0",
				"v0.15.0",
				"v0.16.0",
				"v0.17.0",
				"v0.18.0",
				"v0.19.0",
				"v0.2.0",
				"v0.20.0",
				"v0.21.0",
				"v0.21.0-rc1",
				"v0.21.0-rc2",
				"v0.22.0",
				"v0.22.0-rc1",
				"v0.22.0-rc2",
				"v0.23.0",
				"v0.23.0-rc1",
				"v0.23.0-rc2",
				"v0.24.0",
				"v0.24.0-rc1",
				"v0.24.4",
				"v0.25.0",
				"v0.25.0-rc1",
				"v0.25.0-rc2",
				"v0.25.1",
				"v0.26.0",
				"v0.26.0-rc1",
				"v0.26.0-rc2",
				"v0.27.0-rc1",
				"v0.27.0-rc2",
				"v0.3.0",
				"v0.8.0"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-8099.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://lists.debian.org/debian-lts-announce/2022/03/msg00031.html"
		},
		{
			"type": "FIX",
			"url": "https://github.com/libgit2/libgit2/commit/58a6fe94cb851f71214dbefac3f9bffee437d6fe"
		},
		{
			"type": "WEB",
			"url": "https://libgit2.github.com/security/"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2018-8099"
		}
	]
}
